src/index.html

```diff
@@ -11,7 +11,143 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+       action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+       method="post"
+      >
```

Comment:
```suggestion
    <form
       action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
       method="post"
     >
```
put closing bracket on the same level as the opening one